helpdesk_mgmt_team_restrict/models/helpdesk_ticket_team.py

```diff
@@ -0,0 +1,18 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import models, api
+
+
+class HelpdeskTicketTeam(models.Model):
+    _inherit = 'helpdesk.ticket.team'
+
+    @api.model
+    def create(self, vals):
+        self.clear_caches()
+        return super().create(vals)
```

Comment:
Not tested, bettet use create multi:
```suggestion
    @api.model_create_multi
    def create(self, vals_list):
        self.clear_caches()
        return super().create(vals_list)
```